pkg/ring/lifecycler.go

```diff
@@ -422,7 +422,10 @@ loop:
 	}
 
 	// Mark ourselved as Leaving so no more samples are send to us.
-	i.changeState(context.Background(), LEAVING)
+	err := i.changeState(context.Background(), LEAVING)
+	if err != nil {
+		level.Error(util.Logger).Log("msg", "failed to set state to LEAVING", "ring", i.RingName, "err", err)
```

Comment:
`to set state` > `to change the state`

Reply:
Sorry Im not following @pracucci ..... are you suggesting I change the reported message?? I followed the lead of what we have here.

https://github.com/cortexproject/cortex/blob/b4a27d9d9732fdc118450bc8b68bfdcc55a3dd10/pkg/ring/lifecycler.go#L400-L403

Reply:
Yes, I was suggesting to change to message but even the current one is fine if already used in other places. Let me resolve this comment.